### PR TITLE
Fix issue #166: Organize parameters in sections and groups

### PR DESCRIPTION
**Pull request by dvandyk**
I kept all parameter names as they are, but we should seriously move to using ```QualifiedName``` also for parameter names.

### CODE REVIEW

File: eos/parameters/sm-and-eft.yaml [hunk 1]
```diff
@@ -0,0 +1,866 @@
...
+          central: 0.0
+          min:     0.0
+          max:     0.0
+      'b->ctaunutau::Im{cT}' :
+          central: 0.0
+          min:     0.0
+          max:     0.0
+  # }}}
+
+  # b -> c l^- nubar WET Parameters
```
Comment (cBobeth) [inline eos/parameters/sm-and-eft.yaml]: Should be "b -> u" instead of "b -> c"

File: eos/parameters/sm-and-eft.yaml [hunk 2]
```diff
@@ -0,0 +1,866 @@
...
+  # {{{
+  - title: '$b\to c\ell^-\bar\nu$ WET Parameters'
```
Comment (cBobeth) [inline eos/parameters/sm-and-eft.yaml]: b->u

File: eos/parameters/sm-and-eft.yaml [hunk 3]
```diff
@@ -0,0 +1,866 @@
...
+    description: >
+        The list of parameters describing the $b\to c\ell^-\bar\nu$ Wilson coefficients.
```
Comment (cBobeth) [inline eos/parameters/sm-and-eft.yaml]: b->u

File: eos/parameters/form-factors.yaml [hunk 4]
```diff
@@ -1,2548 +1,2558 @@
...
-    central: 0.0
-    min:     0.0
-    max:     0.0
-"B->pipi::c^Ftime_1_2@FvDV2018" :
-    central: 0.0
-    min:     0.0
-    max:     0.0
+  # Parameters in LCSR Calculations with Pion LCDAs
+  # {{{
+  - title: 'Parameters in LCSR Calculations with Pion LCDAs'
```
Comment (cBobeth) [inline eos/parameters/form-factors.yaml]: Pions and also K^* = light meson LCDA's ?

**Comment by dvandyk** [comment]
@cBobeth thanks!

**Pull request merged**